coalib/collecting/Dependencies.py

```diff
@@ -23,7 +13,7 @@ def _resolve(bears, resolved_bears, seen):
 
         if bear in seen:
             seen.append(bear)
-            raise CircularDependencyError.for_bears(seen)
+            raise CircularDependencyError([s.name for s in seen])
```

Comment:
I think you don't have to collect the generator in a list before ;) so just
```python
raise CircularDependencyError(s.name for s in seen)
```
should suffice ;)